docs/getting-started/install-configure.md

```diff
@@ -5,16 +5,39 @@ weight: 2
 indent: true
 ---
 
-# Install & Configure Crossplane
+# Choosing Hosted vs. Self-Hosted Crossplane
 
-Crossplane can be easily installed into any existing Kubernetes cluster using
-the regularly published Helm chart. The Helm chart contains all the custom
-resources and controllers needed to deploy and configure Crossplane.
+Users looking to use Crossplane for the first time have two options avaiable to them 
+today. The easiest way to start using Crossplane quickly is to use a hosted Crossplane 
+service like Upbound Cloud. Setup should take 10-15 minutes total.
 
-See [Install] and [Configure] docs for installing alternate versions and more
-detailed instructions.
+Alternatively, users looking for some more flexability can install Crossplane into their 
+own Kubernetes cluster. Crossplane will be installed using the regularly published Helm 
+chart. The Helm chart contains all the custom resources and controllers needed to deploy 
+and configure Crossplane.
+
+Users choosing the self-hosted option can reference our [Install] and [Configure] docs 
+for installing alternate versions and more detailed instructions.
+
+## Start with a Hosted Crossplane
+Upbound Cloud is a managed service of Crossplane created by the founders of Crossplane. 
+You can [create an account](https://cloud.upbound.io/register) to get started. Once 
+logged in, you can [create](https://cloud.upbound.io/docs/getting-started/set-up-upbound-cloud) 
+and then [connect](https://cloud.upbound.io/docs/getting-started/connect-to-your-platform) 
+to your hosted Crossplane cluster.
 
-## Get a Kubernetes Cluster
+Once you've completed thse two steps, skip down to 
+[Install Crossplane CLI](#install-crossplane-cli) for further setup instructions.
+
+<i>Want see another hosted Crossplane service listed? Please 
+<a href="http://slack.crossplane.io/">reach out on Slack</a> and our community will 
+highlight it here!</a></i>
```

Comment:
I think you can use markdown for this, i.e.

```markdown
Want see another hosted Crossplane service listed? Please [reach out on Slack][Slack] and our community will 
highlight it here!
````

Note that you'll need to add `[Slack]: http://slack.crossplane.io/` to the links at the bottom.